docs/data/install-xrootd.md

```diff
@@ -287,13 +287,32 @@ fermicloud054.fnal.gov complete inventory as of Tue Apr 12 07:38:29 2011 /data/x
 
 **Note**: In this example, `fermicloud53.fnal.gov` is a redirector node and `fermicloud054.fnal.gov` is a data node.
 
+### (Optional) Enabling a FUSE mount
+
+You can use a FUSE mount in order to test POSIX access to xrootd in the GridFTP server. Configure it using the following steps.
+
+Modify **`/etc/fstab`** by adding the following entries:
+
+    :::file
+    ....
+    xrootdfs                %RED%/mnt/xrootd%ENDCOLOR%              fuse    rdr=xroot://%RED%redirector1.domain.com%ENDCOLOR%:1094/%RED%/path/%ENDCOLOR%,uid=xrootd 0 0
+
+
+Replace `/mnt/xrootd` with the path that you would like to access with. This should also match the GridFTP settings for the `XROOTD_VMP` local path. Create **`/mnt/xrootd`** directory. Once you are finished, you can mount it:
+
+    :::file
+    mount /mnt/xrootd
+
+You should now be able to run UNIX commands such as `ls /mnt/xrootd` to see the contents of the XRootD server.
+
 ### (Optional) Authorization
 
 There are several authorization options in XRootD available through the security
 plugins. In this document, we will cover two options for security:
 
 -   Simple Unix Security: Based on user accounts that the client is logged in as.
--   xrootd-lcmaps: Using the lcmaps callout to use GUMS authorization
+-   Shared-key based authentication.
+-   `xrootd-lcmaps`: Using the LCMAPS callout to use X509-based authentication
```

Comment:
Let's add links to the respective sub-sections for each of these options

Reply:
Done.  This is a bit of mess though, since I'm not sure `xrootd-lcmaps` is really equivalent to the others.  Not touching it right now.